src/sort/simple-sort-directive.js

```diff
@@ -0,0 +1,239 @@
+/**
+ * @ngdoc directive
+ * @name patternfly.sort.directive:pfSimpleSort
+ *
+ * @description
+ *   Directive for a simple sort component
+ *   <br><br>
+ *
+ * @param {object} config configuration settings for the sort:<br/>
+ * <ul style='list-style-type: none'>
+ * <li>.fields          - (Array) List of sortable fields containing:
+ * <ul style='list-style-type: none'>
+ * <li>.id          - (String) Unique Id for the sort field
+ * <li>.title       - (String) The title to display for the sort field
+ * <li>.sortType    - (String) The sort type, 'alpha' or 'numeric'
+ * </ul>
+ * <li>.sortId   - (Object) Id of the current sort field
+ * <li>.sortDirection - (String) The current sort direction 'asc' or 'desc'
+ * <li>.onSortChange - ( function(sortId, sortDirection ) Function to call when the current sort params change
```

Comment:
This is by design. This describes the config object so this is trying to show the fields for the object.